Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -66,15 +66,18 @@ enum PackageController {
         case img
         case index
         case js
+        case linkableEntities = "linkable-entities.json"
         case themeSettings = "theme-settings.json"
         case tutorials
 
         var contentType: String {
             switch self {
                 case .css:
                     return "text/css"
-                case  .data, .faviconIco, .faviconSvg, .images, .img, .index, .themeSettings:
```

Comment:
I tossed them all into `application/octet-stream` because these files aren't really "served" in the sense that anything other than the DocC JS ingests them and that doesn't seem to care about content-types.

I guess it's fine to adjust it but the bigger question is why we should proxy linkable entities (which I suppose triggered this change). Is the DocC site using this in some way?